dcrtimed/backend/filesystem/filesystem.go

```diff
@@ -748,6 +749,81 @@ func (fs *FileSystem) GetTimestamps(timestamps []int64) ([]backend.TimestampResu
 	return gtmes, nil
 }
 
+// Get the last n digests in the added to the Backend
+func (fs *FileSystem) LastDigests(n int32) ([]backend.GetResult, error) {
+	files, err := ioutil.ReadDir(fs.root)
```

Comment:
This is deprecated.  Use os.ReadDir instead

https://pkg.go.dev/io/ioutil#ReadDir